README.md

```diff
@@ -129,19 +129,24 @@ The are numbers of ways you can run Unleash.
 
 ### Run from Docker
 
-1. Create a network by running docker network create unleash
+1. Create a network by running the following command:
+
+```sh
+docker network create unleash
+```
+
 2. Start a postgres database:
 
 ```sh
-docker run -e POSTGRES_PASSWORD=some_password \
+docker run -d -e POSTGRES_PASSWORD=some_password \
```

Comment:
For the readme, we'd like to keep the command examples blocking. This makes it easier for people who aren't used to Docker to get started. See my general review comment for a more thorough discussion ☺️ 

```suggestion
docker run -e POSTGRES_PASSWORD=some_password \
```